src/gallery/slider.js

```diff
@@ -79,12 +79,14 @@ const clientWidth = window.innerWidth,
 			},
 
 			author = mediaInfo.author ? mediaInfo.author : msg( lang, 'gallery-unknown-author' ),
-			link = mediaInfo.filePage
+			link = mediaInfo.filePage,
+			description = getImageDescription()
 
 		// @todo consider a wrapper container for all the image info?
 		return `
 			<div class="${prefixClassname}-item-caption">
-				<div class="${prefixClassname}-item-caption-text">${getImageDescription()}</div>
+				${description.length > 95 ? `<div class="${prefixClassname}-item-caption-expand-cue"></div>` : ''}
```

Comment:
An idea to roughly estimate the number of lines the caption has.

Reply:
Did you test it with languages using other writing systems (Chinese, Arabic, Thai, Hebrew, etc)?

Reply:
I tested with Arabic and Thai from our demo site and seems to work fine. Indeed different languages behave slightly different and that's why using string length gives us only a rough estimate, but seems this could work. Still need to hear Sudhanshu's thoughts and reaction though. The `95` value is something we can tweak and fine-tune.

Reply:
Synced with Sudhanshu today, I'm introducing https://github.com/wikimedia/wikipedia-preview/pull/85/commits/0a7e5f6ad56801bb5fb169f1879990a16611d4a0 to refine this a bit. Idea is to use both string length and screen width size to gauge better if the expand cue should be rendered or not. Will continue syncing/refining with him tomorrow.

Reply:
> Idea now is to use both string length and screen width size to gauge

it's actually look good to me with the magic number, if this pass the design, then I'm fine with it.